tools/CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ function(pico_init_picotool)
         endif()
     endif()
 
+    if (TARGET picotool AND NOT DEFINED ENV{_PICOTOOL_FOUND_THIS_RUN})
```

Comment:
Should this also check `picotool_FOUND` ?

Reply:
I don't think it needs to - if the `picotool` target has been created, then it should work fine